unified-runtime/source/loader/layers/sanitizer/msan/msan_buffer.cpp

```diff
@@ -63,38 +59,11 @@ ur_result_t EnqueueMemCopyRectHelper(
   // loop call 2D memory copy function to implement it.
   for (size_t i = 0; i < Region.depth; i++) {
     ur_event_handle_t NewEvent{};
-    UR_CALL(getContext()->urDdiTable.Enqueue.pfnUSMMemcpy2D(
+    UR_CALL(msan::urEnqueueUSMMemcpy2D(
```

Comment:
Just comment: using APIs from msan_ddi so that the shadow and origin can be copied automatically